src/scripts/main.js

```diff
@@ -1,3 +1,27 @@
 'use strict';
+debugger;
 
-// write code here
+function getEmployees(list = document.getElementsByTagName('li')) {
+  const people = [...list].map(function (item) {
```

Comment:
```suggestion
  const people = [...list].map((item) => {
```
Используй стрелочные функции